<definition xmlns="http://oval.mitre.org/XMLSchema/oval-definitions-5" class="patch" id="oval:org.cisecurity:def:6166" version="7">
  <metadata>
    <title>DSA-4476-1 -- python-django -- security update</title>
    <affected family="unix">
      <platform>Debian 9</platform>
      <product>python-django</product>
    </affected>
    <reference ref_id="DSA-4476-1" ref_url="https://www.debian.org/security/dsa-4476" source="VENDOR" />
    <reference ref_id="CVE-2019-6975" ref_url="http://cve.mitre.org/cgi-bin/cvename.cgi?name=CVE-2019-6975" source="CVE" />
    <reference ref_id="CVE-2019-12308" ref_url="http://cve.mitre.org/cgi-bin/cvename.cgi?name=CVE-2019-12308" source="CVE" />
    <reference ref_id="CVE-2019-12781" ref_url="http://cve.mitre.org/cgi-bin/cvename.cgi?name=CVE-2019-12781" source="CVE" />
    <description>Three security issues were found in Django, a Python web development framework, which could result in denial of service,
												incomplete sanitisation of clickable links or missing redirects of HTTP requests to HTTPS.</description>
    <oval_repository>
      <dates>
        <submitted date="2019-07-19T14:31:45">
          <contributor organization="ALTX-SOFT">Alexandr Ruchkin</contributor>
        </submitted>
        <status_change date="2019-07-26T12:00:00.000-05:00">DRAFT</status_change>
        <status_change date="2019-08-09T12:00:00.000-05:00">INTERIM</status_change>
        <status_change date="2019-08-23T12:00:00.000-05:00">ACCEPTED</status_change>
      </dates>
      <status>ACCEPTED</status>
      <min_schema_version>5.10</min_schema_version>
    </oval_repository>
  </metadata>
  <criteria>
    <extend_definition comment="Debian 9 is installed" definition_ref="oval:org.cisecurity:def:2515" />
    <criterion comment="python-django is earlier than 1:1.10.7-2+deb9u5" test_ref="oval:org.cisecurity:tst:10316" />
  </criteria>
</definition>
